### ISSUE TITLE
TYPO3 10.x compatibility 

### ISSUE THREAD

**Issue by instruccionesaldorso** [pixelant/pxa_cookie_bar#16]
Is TYPO3 v10 compatibility planned for this extension?
We'd love to keep using it over our updated instances!

**Comment by dietantejensen** [comment]
Well, as this issue is not even assigned after 6 months, I suppose not and we, too, will have to switch to implementing some js. What, with all the updating and stuff, might be less work anyway. Thaks nonetheless for your work.

**Comment by weisgerberm** [comment]
I added 10.x compatibility with a pull request. If the pull request is not assigned, you can also use my fork here:
#https://github.com/weisgerberm/pxa_cookie_bar

**Comment by instruccionesaldorso** [comment]
@weisgerberm Which branch should we use from your git repo?
"master" branch should include the new 10.4 version constraint here: https://github.com/weisgerberm/pxa_cookie_bar/blob/master/composer.json#L15
besides the ext_emconf.php version increased.

**Comment by weisgerberm** [comment]
> @weisgerberm Which branch should we use from your git repo? "master" branch should include the new 10.4 version constraint here: https://github.com/weisgerberm/pxa_cookie_bar/blob/master/composer.json#L15 besides the ext_emconf.php version increased.

@instruccionesaldorso You're right, I forgot to increase the version number in the composer.json. Its fixed now in the master branch, thanks
